src/app/app-platform/ContentApp.h

```diff
@@ -94,11 +96,30 @@ class ContentAppClientCommandSender
 class DLL_EXPORT ContentApp
 {
 public:
+
+    struct SupportedCluster {
+        int clusterIdentifier { -1 };
+        int features { -1 };
+        std::vector<int> optionalCommandIdentifiers;
+        std::vector<int> optionalAttributesIdentifiers;
```

Comment:
Neither attribute nor command IDs can all be expressed in `int`.  Please use the right types.

Reply:
Will do!